packages/commonjs/test/snapshots/function.js.md

```diff
@@ -215,6 +215,2082 @@ Generated by [AVA](https://ava.li).
+    		else␊
+    		  joined += '/' + arg;␊
+    	  }␊
+    	}␊
+    	if (joined === undefined)␊
+    	  return '.';␊
+      ␊
+    	return joined;␊
+    }␊
+    ␊
+    function isPossibleNodeModulesPath (modulePath) {␊
+    	let c0 = modulePath[0];␊
+    	if (c0 === '/' || c0 === '\\\\') return false;␊
+    	let c1 = modulePath[1], c2 = modulePath[2];␊
+    	if ((c0 === '.' && (!c1 || c1 === '/' || c1 === '\\\\')) ||␊
+    		(c0 === '.' && c1 === '.' && (!c2 || c2 === '/' || c2 === '\\\\'))) return false;␊
+    	if (c1 === ':' && (c2 === '/' || c2 === '\\\\'))␊
+    		return false;␊
+    	return true;␊
+    }␊
+    ␊
+    function commonjsRequire (path, originalModuleDir) {␊
+    	const shouldTryNodeModules = isPossibleNodeModulesPath(path);␊
+    	path = normalize(path);␊
+    	let relPath;␊
+    	while (true) {␊
+    		if (!shouldTryNodeModules) {␊
+    			relPath = originalModuleDir ? normalize(originalModuleDir + '/' + path) : path;␊
+    		} else if (originalModuleDir) {␊
+    			relPath = normalize(originalModuleDir + '/node_modules/' + path);␊
+    		} else {␊
+    			relPath = normalize(join('node_modules', path));␊
+    		}␊
+    		for (let extensionIndex = 0; extensionIndex < CHECKED_EXTENSIONS.length; extensionIndex++) {␊
+    			const resolvedPath = relPath + CHECKED_EXTENSIONS[extensionIndex];␊
+    			let cachedModule = DYNAMIC_REQUIRE_CACHE[resolvedPath];␊
+    			if (cachedModule) return cachedModule.exports;␊
+    			const loader = DYNAMIC_REQUIRE_LOADERS[resolvedPath];␊
+    			if (loader) {␊
+    				DYNAMIC_REQUIRE_CACHE[resolvedPath] = cachedModule = {␊
+    					id: resolvedPath,␊
+    					filename: resolvedPath,␊
+    					exports: {},␊
+    					parent: DEFAULT_PARENT_MODULE,␊
+    					loaded: false,␊
+    					children: [],␊
+    					paths: []␊
+    				};␊
+    				try {␊
+    					loader.call(commonjsGlobal, cachedModule, cachedModule.exports);␊
+    				} catch (error) {␊
+    					delete DYNAMIC_REQUIRE_CACHE[resolvedPath];␊
+    					throw error;␊
+    				}␊
+    				cachedModule.loaded = true;␊
+    				return cachedModule.exports;␊
+    			}		}␊
+    		if (!shouldTryNodeModules) break;␊
+    		const nextDir = normalize(originalModuleDir + '/..');␊
+    		if (nextDir === originalModuleDir) break;␊
+    		originalModuleDir = nextDir;␊
+    	}␊
+    	return require(path);␊
+    }␊
+    ␊
+    commonjsRequire.cache = DYNAMIC_REQUIRE_CACHE;␊
+    ␊
+    var _commonjsHelpers = /*#__PURE__*/Object.freeze({␊
+    	__proto__: null,␊
+    	commonjsGlobal: commonjsGlobal,␊
+    	unwrapExports: unwrapExports,␊
+    	createCommonjsModule: createCommonjsModule,␊
+    	getCjsExportFromNamespace: getCjsExportFromNamespace,␊
+    	commonjsRegister: commonjsRegister,␊
+    	commonjsRequire: commonjsRequire␊
+    });␊
+    ␊
+    _commonjsHelpers.commonjsRegister("/Users/lukastaegert/test/dynamic-require-plugins/packages/commonjs/test/fixtures/function/dynamic-require/submodule.js", function (module, exports) {␊
```

Comment:
This is not acceptable to me. IMO is no reason to use absolute paths. Instead, you could determine the common root directory of all dynamic require targets and make all paths relative to that. E.g. use https://www.npmjs.com/package/commondir

Reply:
I thought about adding such a function, but did not want to make the PR larger than it needs to be, or add more dependencies. If you're OK with this, then I'll roll with you.

Anyway the comment in the docs should stay, as the "common dir" could still expose private directory names.